src/enable_variable_ops/ngraph_encapsulate_op.cc

```diff
@@ -172,22 +173,17 @@ class NGraphEncapsulateOp : public OpKernel {
       // m_freshness_tracker->Unref();
     }
 
-    string node_name = "_ngraph_cluster_" + to_string(m_ngraph_cluster);
-    // TODO(malikshr) : Could be erroreneous we dont know if this
```

Comment:
There are only two main changes. Other diffs are because of logs that i added. I will point them out.
The key being generated was wrong due to the string above.